lib/ui/worker/cache.dart

```diff
@@ -531,6 +531,19 @@ class CacheWorker extends DisposableService {
           );
     }
   }
+
+  /// Opens a [File] or a directory containing it.
+  Future<void> _openDirectoryOrFile(File file) async {
+    if (Platform.isWindows) {
```

Comment:
Shouldn't `PlatformUtils.isWindows` be used instead? But it would also be required to do before every other `else`:

```
if (PlatformUtils.isWeb) {
  return;
}
```